app/imports/startup/client/routes/posts.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { FlowRouter } from 'meteor/kadira:flow-router';
+import { mount } from 'react-mounter';
+
+import { MainContainer } from '/imports/containers/MainContainer.jsx';
+
+import { NewPost } from '/imports/components/posts/NewPost.jsx';
+
+// Post Routes
+const postsRoutes = FlowRouter.group({
+  name: 'posts',
+});
+
+postsRoutes.route('/new', {
```

Comment:
refactor this to use session variables rather than the query params. route back to home if not set on /new (for now until supporting file input on that view)